lib/fizzy/parser.cpp

```diff
@@ -12,6 +12,22 @@
 
 namespace fizzy
 {
+namespace
+{
+void validate_constant_expression(const ConstantExpression& const_expr, const Module& module)
+{
+    if (const_expr.kind == ConstantExpression::Kind::Constant)
+        return;
+
+    const auto global_idx = const_expr.value.global_index;
+    if (global_idx >= module.get_global_count())
+        throw validation_error{"invalid global index in constant expression"};
+
+    if (module.is_global_mutable(global_idx))
+        throw validation_error("constant expression can use global.get only for const globals");
```

Comment:
Why is this using `validation_error()` while the other `validation_error{}` ?

Reply:
Changed it to always curly braces in my changes.